src/site/_includes/components/Meta.js

```diff
@@ -130,6 +135,31 @@ module.exports = (locale, page, collections, renderData = {}) => {
     `;
   }
 
+  function renderHreflangMeta() {
+    if (!pageUrl) {
+      return;
+    }
+    const url = pageUrl.startsWith('/i18n/')
+      ? pageUrl.replace(i18nRegex, '')
+      : pageUrl;
+
+    // Find i18n equivalents of the current url and heck if they exist.
+    const langhrefs = i18nLocales
+      .map((locale) => [locale, `/i18n/${locale}${url}`])
+      .filter((langhref) => findByUrl(langhref[1]))
```

Comment:
I was a little confused by this because I thought calling `findByUrl` on something that hasn't been translated would either throw an error or return undefined. I didn't realize that Array.filter() coerces undefined to `false`, so it's being used intentionally here.

Would you mind adding a comment above line 149 saying something like "If a translated version of the page does not exist then findByUrl will return undefined which coerces to false and removes the untranslated page from the array. By the end we have an array that only contains translated pages."